### DISCUSSION TITLE
my modded minecraft crahs when  I launch it

### DISCUSSION THREAD

**Discussion by KAARVAAH** [FabricMC/fabric-api#3430] [Player support]
### Minecraft Logs

```txt
log: https://mclo.gs/YNWArqq
```

### Description

When  I  launch the game it try to, but it can't and crashes

**Comment by KAARVAAH** [comment]
someone could  help me please?

**Reply by Linguardium** [reply]
do you have the crashlog from the crash-reports folder?

**Reply by KAARVAAH** [reply]
no man, idk why, but my launcher ? i think is who doesn't make the crashlogs

**Reply by KAARVAAH** [reply]
but only today because past days  it worked properly doing it

**Reply by Linguardium** [reply]
replace the launcher? try prismlauncher and see if you still have issues

**Reply by KAARVAAH** [reply]
[crash-2023-11-19_20.56.27-client.txt](https://github.com/FabricMC/fabric/files/13405484/crash-2023-11-19_20.56.27-client.txt)
sorry for the delay

**Reply by Linguardium** [reply]
does this happen every launch? if so, it appears to be an issue with the small ships mod config file

**Reply by KAARVAAH** [reply]
man I deleted smallships and its config and it still crashing in every launch, both launchers
